### DISCUSSION TITLE
Standalone tool to convert `.norg` to `.md`

### DISCUSSION THREAD

**Discussion by ssiyad** [nvim-neorg/neorg#1297] [Q&A] [answered]
Is there any standalone/external tool too convert `.norg` files into `.md`? For example, I want to generate a `readme.md` whenever `readme.norg` is updated, using github ci. Is such a tool already available, or is it in plans?

Assuming there isn't such tool, how can I write one? Is there a neorg spec I can refer?

**Comment by pysan3** [comment, marked as answer]
https://github.com/pysan3/Norg-Tutorial/blob/main/norg_tutorial.md#export--import

Also there's https://github.com/VoreckLukas/norg-pandoc-rs which is still WIP but he said he'll make an action when he's done.

**Reply by pysan3** [reply]
Btw nothing is stopping you from using neovim as a lua interpreter in a GH action lol. You can test any neovim plugin and use the `vim.*` apis using this method.

**Reply by ssiyad** [reply]
I just did not want to use neovim lol. But this works for now ig. Thanks!

**Reply by pysan3** [reply]
It's at least way faster to use external binaries anyways lol

If you make a GH action please share it with the community. I think everybody is waiting for it.
